File: app/src/main/java/com/visiontutor/app/AddReview.java
package com.visiontutor.app;

import android.app.Dialog;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;

public class AddReview extends AppCompatActivity {

    Button b1;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_addreview);

        b1 = findViewById(R.id.reviewproceed);

        b1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Dialog dialog = new Dialog(AddReview.this);
                dialog.setContentView(R.layout.reviewpopup);
                dialog.show();
            }
        });

    }
}

File: app/src/main/java/com/visiontutor/app/AddReview.kt
package com.visiontutor.app

import android.app.Dialog
import android.os.Bundle
import android.widget.Button

import androidx.appcompat.app.AppCompatActivity

class AddReview : AppCompatActivity() {

    lateinit var b1: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_addreview)

        b1 = findViewById(R.id.review_proceed)

        b1.setOnClickListener {
            val dialog = Dialog(this)
            dialog.setContentView(R.layout.reviewpopup)
            dialog.show()
        }

    }
}
